8.cpp

```diff
@@ -4,11 +4,11 @@
 using namespace std;
```

Comment:
I suggest remove this line, and use `std::` instead.

Reply:
already modify